src/region_helper.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*-coding:utf-8-*-
+
+import logging
+
+
+def _found_region(cookies):
+    try:
+        for cookie in cookies:
+            if cookie['name'] == 'JSESSIONID':
+                _region = cookie['domain'].split('.')[0]
+                # 4th region - chinese uses different endpoints, not covered by current plugin
+                if _region.lower() in ['eu', 'us', 'kr']:
+                    return _region
+                else:
+                    raise ValueError(f'Unknown region {_region}')
+        else:  # for
+            raise ValueError(f'JSESSIONID cookie not found')
+    except Exception:
+        return 'eu'
+
+
+def guess_region(local_client):
+    """
+    1. read the consts.py
+    2. try read the battlenet db OR config get the region info.
+    3. failed return ""
+    """
+    from consts import REGION
+    if REGION:
+        return REGION
+
+    try:
+        if local_client._load_local_files():
+            if local_client.config_parser.region:
+                return local_client.config_parser.region.lower()
+
+            if local_client.database_parser.region:
+                return local_client.database_parser.region.lower()
+
+    except Exception as e:
+        logging.error(f'{e}')
+        return ""
```

Comment:
I was thinking about this flow for a while and in the first place it looks pretty solid:

- as it will work for most people without requiring any manual input 
- for those who has no blizzard installed, there is possibility to edit consts.py

But maybe is a better alternative that will work in 100% without battle.net installed?

Did you check response from authorization?
see this line: https://github.com/bartok765/galaxy_blizzard_plugin/pull/9/files#diff-36ba37277336f18cb4e25f67b470135dR53

I don't now we can rely on this, but even if no, after logging on blizzard.com `https://www.blizzard.com/en-us/user` GET is send with region in response  (checking in chrome devtools-Network). Is it the same for China blizzard?

Reply:
This url is helpful. rightnow it's response is :
```
{
  "locale": "en_US",
  "localeRegion": "us",
  "region": "cn",
  "authenticated": 0,
  "userId": null
}
```
I am not login, and It's tell me the region `cn` ，If I through the proxy , the region has changed to `us` . what's your opnion about rely on the url response as a second choice option.

Reply:
@bartok765 all have done. Still in my machine i need the _psutil_windows.cp37-win32.pyd, it's has been removed,  this is problem i hope you give some thought on it.